README.md

```diff
@@ -208,30 +115,12 @@ Example:
 
 > Mention all of the collaborators of this project.
```

Comment:
- [ ] Kindly remove all the descriptive section on your README. Leaving it there does not make your README file look professional.